### DISCUSSION TITLE
why my train is 0 ,it isn't move;

### DISCUSSION THREAD

**Discussion by kinghge** [M-3LAB/open-iad#5] [Q&A]
---------------------
{'paradigm': 'c2d', 'dataset': 'mvtec2d', 'model': 'cfa', 'net': 'net_cfa', 'train_task_id': [0], 'valid_task_id': [0], 'vanilla': True, 'gpu_id': '0', 'vis': True}
---------------------
{
    'dataset': 'mvtec2d',
    'root_path': 'D:\\DATA',
    'data_path': 'D:\\DATA/mvtec2d',
    'num_task': 15,
    'data_size': 256,
    'data_crop_size': 256,
    'mask_size': 256,
    'mask_crop_size': 256,
    'learning_mode': 'centralized',
    'vanilla': True,
    'semi': False,
    'semi_anomaly_num': 5,
    'semi_overlap': False,
    'ref_num': 5,
    'continual': False,
    'fewshot': False,
    'fewshot_exm': 5,
    'fewshot_num_dg': 4,
    'fewshot_data_aug': False,
    'fewshot_feat_aug': False,
    'fewshot_aug_type': ['normal'],
    'noisy': False,
    'noisy_ratio': 0.1,
    'noisy_overlap': False,
    'transfer': False,
    'transfer_target_sample_num': 8,
    'gpu_id': '0',
    'gpu_ids': ['000000'],
    'num_workers': 8,
    'work_dir': './work_dir',
    'save_log': True,
    'debug': 'true pytorch',
    'batch_limit': 2,
    'seed': 66,
    'train_task_id': [0],
    'valid_task_id': [0],
    'train_task_id_tmp': 10,
    'valid_task_id_tmp': 10,
    'vis': True,
    'vis_num': 10,
    'vis_em': False,
    'server_moda': 'eno1',
    'model': 'cfa',
    'net': 'net_cfa',
    'num_epochs': 50,
    'train_batch_size': 10,
    'valid_batch_size': 10,
    'train_aug_type': 'normal',
    'valid_aug_type': 'normal',
    'gamma_c': 1,
    'gamma_d': 1,
    'file_path': './work_dir/centralized/mvtec2d/11'
}
---------------------
-> train ...
run task: 0, bottle
